yarn.lock

```diff
@@ -8804,7 +8775,7 @@ simple-swizzle@^0.2.2:
   dependencies:
     is-arrayish "^0.3.1"
 
-sinon@^9.0.1, sinon@^9.0.2:
+sinon@^9.0.1:
```

Comment:
just a minor: the two sinon versions could both be covered with 9.2.3, you could run yarn-deduplicate over the file to remove the obsolete

Reply:
done